schutzbot/scheduled_cloud_cleaner.sh

```diff
@@ -179,11 +179,31 @@ for snapshot in ${SNAPSHOTS}; do
 	START_TIME=$(echo "${snapshot}" | jq '.StartTime' | tr -d '"')
 
 	if [[ ${TAG} == "gitlab-ci-test" && ${TAG_VALUE} == "true" ]]; then
-		if [[ $(date -d "${START_TIME}" +%s) -lt $(date -d "- 2 minutes" +%s) ]]; then
+		if [[ $(date -d "${START_TIME}" +%s) -lt "${DELETE_TIME}" ]]; then
 			$AWS_CMD ec2 delete-snapshot --snapshot-id "${SNAPSHOT_ID}"
 			echo "The snapshot with id ${SNAPSHOT_ID} was deleted"
 		else
 			echo "The snapshot with id ${SNAPSHOT_ID} was created less than ${HOURS_BACK} hours ago"
 		fi
 	fi
 done
+
+# Remove tagged and old enough s3 objects
+AWS_BUCKET=composerredhat
```

Comment:
I'm sorry I just noticed this. Why are you defining this variable here? It's defined in our CI variables and if someone changes it, the s3 test will use a new bucket while this will keep cleaning this hard coded bucket :/ .. I also noticed that this is missing the `greenprint` function that is used on line 93, I should have noticed in the previous PR. You can add this here if you want it to work:
```
 # Colorful output.                                                                                                                                                                                                 
  function greenprint {                                                                                                                                                                                              
      echo -e "\033[1;32m[$(date -Isecond)] ${1}\033[0m"                                                                                                                                                             
  }
```

Reply:
Whops, forgot to remove that variable used for testing, thanks. I didn't notice greenprint was not defined, as it is used before on the script, I'll add it. Thanks